src/shared/user-menu.jsx

```diff
@@ -52,6 +53,20 @@ export default class UserMenu extends React.Component {
     this.setState({ loginModalVisible: true })
   }
 
+  logoutSuccess() {
+    if (this.props.refreshOnLoginLogout) {
+      window.location.reload()
+    }
+    // else...
+    this.setState({ isLoggedIn: false })
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  logoutError(response) {
```

Comment:
It is ok just to use an anonymous function when something is so short, instead of disabling eslint.

e.g.

```js
// FIXME: we should handle the logout failure case somehow (e.g. by popping up a notification)
logout(this.logoutSuccess, (response) => console.error('Logout unsuccessful', response))
```